airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -17,16 +17,16 @@
         -->
 
         <!-- Always log INFO by default. -->
-        <Property name="log-level">$${sys:LOG_LEVEL:-${env:LOG_LEVEL:-INFO}}</Property>
+        <Property name="log-level">${sys:LOG_LEVEL:-${env:LOG_LEVEL:-INFO}}</Property>
 
-        <Property name="s3-bucket">$${sys:S3_LOG_BUCKET:-${env:S3_LOG_BUCKET}}</Property>
-        <Property name="s3-region">$${sys:S3_LOG_BUCKET_REGION:-${env:S3_LOG_BUCKET_REGION}}</Property>
-        <Property name="s3-aws-key">$${sys:AWS_ACCESS_KEY_ID:-${env:AWS_ACCESS_KEY_ID}}</Property>
-        <Property name="s3-aws-secret">$${sys:AWS_SECRET_ACCESS_KEY:-${env:AWS_SECRET_ACCESS_KEY}}</Property>
-        <Property name="s3-minio-endpoint">$${sys:S3_MINIO_ENDPOINT:-${env:S3_MINIO_ENDPOINT}}</Property>
-        <Property name="s3-path-style-access">$${sys:S3_PATH_STYLE_ACCESS:-${env:S3_PATH_STYLE_ACCESS}}</Property>
+        <Property name="s3-bucket">${sys:S3_LOG_BUCKET:-${env:S3_LOG_BUCKET}}</Property>
```

Comment:
@jrhizor looks like we were using the wrong syntax here.

Reply:
Double means it's evaluated at log time instead of config time, which is necessary for the sys vars I believe (maybe this is broken in the upgrade somehow)